config/subsidies-consumer/subsidy-attachments-dispatching/config.js

```diff
@@ -3,10 +3,10 @@ const MU_CALL_SCOPE_ID_INITIAL_SYNC = process.env.MU_CALL_SCOPE_ID_INITIAL_SYNC
 const BYPASS_MU_AUTH_FOR_EXPENSIVE_QUERIES = process.env.BYPASS_MU_AUTH_FOR_EXPENSIVE_QUERIES == 'true' ? true : false;
 const DIRECT_DATABASE_ENDPOINT = process.env.DIRECT_DATABASE_ENDPOINT || 'http://virtuoso:8890/sparql';
 const MAX_DB_RETRY_ATTEMPTS = parseInt(process.env.MAX_DB_RETRY_ATTEMPTS || 5);
-const SLEEP_BETWEEN_BATCHES = parseInt(process.env.SLEEP_BETWEEN_BATCHES || 1000);
+const SLEEP_BETWEEN_BATCHES = parseInt(process.env.SLEEP_BETWEEN_BATCHES || 1000); // 1 second
 const MAX_FILE_DOWNLOAD_RETRY_ATTEMPTS = parseInt(process.env.MAX_FILE_DOWNLOAD_RETRY_ATTEMPTS || 5);
-const SLEEP_TIME_AFTER_FAILED_FILE_DOWNLOAD_OPERATION = parseInt(process.env.SLEEP_TIME_AFTER_FAILED_FILE_DOWNLOAD_OPERATION || 1000);
-const SLEEP_TIME_AFTER_FAILED_DB_OPERATION = parseInt(process.env.SLEEP_TIME_AFTER_FAILED_DB_OPERATION || 60000);
+const SLEEP_TIME_AFTER_FAILED_FILE_DOWNLOAD_OPERATION = parseInt(process.env.SLEEP_TIME_AFTER_FAILED_FILE_DOWNLOAD_OPERATION || 60000); // 1 minute
```

Comment:
It's a detail on naming, but maybe `INITIAL_SLEEP_TIME_AFTER_FAILED_FILE_DOWNLOAD_OPERATION` as now we're doubling it every time we retry?